ios-base/AppDelegate.swift

```diff
@@ -50,7 +49,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     //Take user to onboarding if needed, do NOT redirect the user if is already in the landing
     // to avoid losing the current VC stack state.
     if window?.rootViewController is HomeViewController {
```

Comment:
It actually depends on the project but this is done this way because failed login and sign up attepmts are also `unauthorized 401`. Doing so will cause you to be redirected to the landing everytime. For the example app I think is better to check for the Home which indicates that you are logged in, that checking that is not the landing, nor the sign in nor the sign up and each other possible view where you can receive the 401 error and it is in the initial storyboard.

Reply:
Perhaps we can refactor so sign in/up don't trigger an `unexpectedLogout` but its not the scope of this PR of course